paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -286,6 +288,20 @@ internal class CustomerSheetViewModel @Inject constructor(
         }
     }
 
+    private fun onAddPaymentMethodItemChanged(paymentMethod: LpmRepository.SupportedPaymentMethod) {
+        updateViewState<CustomerSheetViewState.AddPaymentMethod> {
+            it.copy(
+                formArguments = createFormArguments(paymentMethod),
+                selectedPaymentMethod = paymentMethod,
+                primaryButtonLabel = if (paymentMethod.code == PaymentMethod.Type.USBankAccount.code) {
+                    resources.getString(com.stripe.android.ui.core.R.string.stripe_continue_button_label)
+                } else {
+                    null
```

Comment:
Should we make this non-nullable and always pass a label to the screen, so that we don’t have to make any decision in the UI code?